src/components/List/List.jsx

```diff
@@ -81,6 +82,10 @@ const propTypes = {
   onItemMoved: PropTypes.func,
   /** callback function when reorder will occur - can cancel the move by returning false */
   itemWillMove: PropTypes.func,
+  /** whether to show content if list is empty */
+  hasEmptyState: PropTypes.bool,
+  /** content shown if list is empty */
+  emptyState: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```

Comment:
`hasEmptyState` seems redundant here. If `hasEmptyState` is removed,`emptyState` can be unconditionally rendered when there are no items.

If someone desires for the empty state to be totally blank, they can configure `emptyState` to be an empty string.

```suggestion
  /** content shown if list is empty */
  emptyState: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```